front/lib/models/assistant/actions/tables_query.ts

```diff
@@ -210,6 +211,10 @@ AgentTablesQueryAction.init(
       type: DataTypes.STRING,
       allowNull: true,
     },
+    functionCallName: {
```

Comment:
For tables query, we will let the user picks a different function name?

Reply:
The function call name will be whatever they decide to name their action in the assistant (like for the other type of actions).

Otherwise if they call it `query_spreadsheet` but we render it in history as `query_tables the model will get confused and try to call a `query_tables` function that doesn't exist

Reply:
Okok definitely makes sense!